src/components/systemButtons/systemButtom.stories.tsx

```diff
@@ -0,0 +1,179 @@
+// import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import Button,{ButtonProps} from './systemButtons';
+
+export default {
+  title: 'components/Button',
+  component: Button,
+  argTypes: {
+    children: { control: 'text' },
+  },
+} as Meta;
+
+const Template: Story<ButtonProps> = (args) => <Button {...args} />;
```

Comment:
Don't show all variants. The idea is to only show one and have pickers for the options (check NavbarStories, see how changing the variant changes the component `options: ["default", "selected", "disabled"]`)